backend/src/routes/units.ts

```diff
@@ -16,4 +17,5 @@ router.get("/:id", UnitController.getUnitHandler);
 
 router.post("/", validateWith(createUnitValidators), UnitController.createUnitsHandler);
 
+router.get("/:id/referrals", requireHousingLocator, UnitController.getUnitReferralsHandler);
```

Comment:
Also, RSs should also be able to see the Referrals table in addition to HLs so I don't think we should restrict this to just HLs.